src/index.js

```diff
@@ -63,12 +63,12 @@ const createList = () => {
 
   trashIcon.addEventListener('click', () => {
     let count = 0;
-    /* eslint-disable */
+    
     form.removeChild(list);
     const getLocal = JSON.parse(localStorage.getItem('list'));
     const data = Array.from(getLocal).filter((i) => i.completed === false);
     data.map((i) => i.index = count += 1);
```

Comment:
You can make use of method chaining here and in other parts of the project to add the map method immediately after the filter method.
This will reduce your code base and make the app more optimized